quixstreams/dataframe/dataframe.py

```diff
@@ -48,6 +48,8 @@
 from quixstreams.processing import ProcessingContext
 from quixstreams.sinks import BaseSink
 from quixstreams.state.types import State
+from quixstreams.sources import BaseSource
+from quixstreams.rowproducer import RowProducer
```

Comment:
Minor nitpick: let's keep this import section sorted.

Reply:
Thanks. Yeah I'd like to introduce a proper linter for that. Like isort or ruff

Reply:
Yes, we need that. For now I use isort separately here and there. We plug it via pre-commit but it is worthy of a separate PR.